packages/website/app/styles/global.css

```diff
@@ -34,46 +40,62 @@ html {
 *::after {
   box-sizing: inherit;
 }
-
 :-moz-focusring {
-  outline: auto;
+  /* outline: auto; */
 }
 
 :focus {
-  outline: var(--color-links) solid 2px;
-  outline-offset: 2px;
+  /* outline: var(--color-links) solid 2px; */
+  /* outline-offset: 2px; */
 }
 
 html,
 body {
   padding: 0;
   margin: 0;
   color: var(--color-foreground);
+  min-height: 100vh;
   background-color: var(--color-background);
+  display: flex;
+  flex-direction: column;
+}
+body::-webkit-scrollbar{
+  display: none;
 }
-
 [data-light] {
```

Comment:
One of the greatest feature of tailwindcss/windicss is https://windicss.org/features/dark-mode.html.

No more explicit css variables are needed.